sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -15,6 +15,23 @@ import { Pipeline } from '@azure/core-rest-pipeline';
 import { RestError } from '@azure/core-rest-pipeline';
 import { TokenCredential } from '@azure/core-auth';
 
+// @public
+export interface AIServicesVisionParameters {
+    apiKey?: string;
```

Comment:
Can you tell me more about this? I see both `apiKey` and `authIdentity` here and I wonder what scenarios do we have where both are provided?